dataland-frontend/src/components/forms/parts/elements/basic/BaseDataPointFormField.vue

```diff
@@ -24,40 +57,72 @@ import { BaseFormFieldProps } from "@/components/forms/parts/fields/FormFieldPro
 import UploadDocumentsForm from "@/components/forms/parts/elements/basic/UploadDocumentsForm.vue";
 import { type DocumentToUpload } from "@/utils/FileUploadUtils";
 import { type BaseDataPoint } from "@/utils/DataPoint";
+import UploadFormHeader from "@/components/forms/parts/elements/basic/UploadFormHeader.vue";
+import { disabledOnMoreThanOne } from "@/utils/FormKitPlugins";
 
 export default defineComponent({
   name: "BaseDataPointFormField",
-  components: { UploadDocumentsForm },
+  components: { UploadFormHeader, UploadDocumentsForm },
   inheritAttrs: false,
-  props: { ...BaseFormFieldProps },
+  props: {
+    ...BaseFormFieldProps,
+    options: {
+      type: Object,
+      require: true,
+    },
+  },
+  inject: {
+    injectlistOfFilledKpis: {
+      from: "listOfFilledKpis",
+      default: [] as Array<string>,
+    },
+  },
   data() {
     return {
+      dataPointIsAvailable: (this.injectlistOfFilledKpis as unknown as Array<string>).includes(this.name as string),
```

Comment:
Double type assertion (`as unknown as Array<string` ) is flakey and bad practice. Why can't we assert `injectlistOfFilledKpis` returns a string Array when defining it?

Reply:
Checked myself. For some reason `this.injectlistOfFilledKpis` is returned as a function and the workaround to get it to be a simple Array<string> type is overkill.